globals/plugin-config.hml

```diff
@@ -0,0 +1,34 @@
+---
+kind: LifecyclePluginHook
+version: v1
+definition:
+  pre: parse
+  name: cache_get_test
+  url:
+    value: http://local.hasura.dev:8787/pre-parse
```

Comment:
Feel like this should be variable in env vars for local, and each of the geographic regions